### ISSUE TITLE
submission

### ISSUE THREAD

**Issue by jackynguyen110** [jackynguyen110/Flick#1]
sorry for not perfect assignment.I will finish it later.This is my submission .Please review.
/cc @coderschoolreview .Thank u very much

**Comment by coderschoolreview** [comment]
:sweat: Sorry but the deadline for submissions has closed. We hope to see you in a future class - to be notified of when new classes open, be sure to like us on facebook at http://www.facebook.com/coderschoolvn.

**Comment by jackynguyen110** [comment]
but the deadline is 6pm sunday

**Comment by jackynguyen110** [comment]
i have some bussiness for company at ha noi.So i must submit my assignment soon

"This week we'll build a Flicks client to display movies currently playing in the theater. The purpose of this project is to develop a basic MVC application and start playing with table views, one of the more complex, but important views in iOS.

Assignment 1: Flicks (Due Sunday, March 13th at 6:00pm)"

Have something wrong here... :(

**Comment by chug2k** [comment]
lol sorry - that is still set from our prework! thank you for submitting your homework early!!

**Comment by chug2k** [comment]
you are the first student to submit homework. thank you again. you are correct, the deadline is 6pm sunday.

**Comment by jackynguyen110** [comment]
Yes.now i must go for my company bussiness to sunday.so i must submit not perfect assignment.sorry for that.thank u very much.

**Comment by minhchau273** [comment]
Fix your `README` with the template on the course portal. Please also try to use the `- [ ]` syntax correctly in the Github flavored markdown syntax. :smile: 
Thanks.

**Comment by harley** [comment]
Hi @jackynguyen110, thanks again for submitting early. Don't forget to fix your README by using the [original file](http://learning.coderschool.vn/snippets/intro_to_ios/readme_templates/flicks_readme.md?raw=true) and mark [x] on the items you completed.

**Comment by jackynguyen110** [comment]
Hi mr.harley anh miss.Minhchau : Sorry for this busy week.I already fixed my readme file.I will promise to finish all function of this assignment today.

**Comment by minhchau273** [comment]
It looks like you still need to complete a required user story for your submission to be complete.
- [ ] User sees an error message when there is a network error.

Please update asap in order for you to receive credit for this assignment.

**Comment by jackynguyen110** [comment]
@minhchau273  sorry for my late.It have done right now.please review for me.

**Comment by minhchau273** [comment]
:+1: Nice work. The point of this homework was to explore a simple example of a full MVC application with a RESTful API. You learned about UITableViews, custom UITableViewCells, navigation from a UITableView, and basic networking.

## Grading Summary:
- **Functionality**: Good
- **Code Style**: Good
- **Visual Design**: Average
- **Usability**: Good

#### **Overall**: Good

## Detailed Notes
- Be sure to follow the Swift code styling conventions for the homework. **INSERT EXAMPLE OF BAD CODE HERE**. Please read this style guide for more detail. Style is important!
  - [Ray Wenderlich Swift Style Guide](https://github.com/raywenderlich/swift-style-guide)
- In future homework, we will cover how to implement models that can deserialize from an NSDictionary. [SwiftyJSON](https://github.com/SwiftyJSON/SwiftyJSON) is a useful project for making it easier to deserialize the JSON response. For a more robust solution, you can also investigate using an ORM like [Pistachio](https://github.com/felixjendrusch/Pistachio).  
- Your custom Movie cell should have a property called `movie`. In the custom setter of `movie`, you should configure the various labels and images. This decouples your custom cells from the table view controllers.
- Note that your images are stretched. You should observe the UIContentMode of the UIImageView. By default, it will stretch the image to match your dimensions, which is probably not what you want.
- Nice work implementing the error message on network failure.
- Nice job implementing a UIScrollView. UIScrollView is the parent class of UITableView.
- Nice job keeping things nice and aligned on a grid. That helps the application look cleaner and make it easier for the user to find the correct information. For a lot more information, you can read [Design tips for iOS9](https://designcode.io/iosdesign-guidelines). 
- The `UISearchBar` doesn't work, right? For now, I cannot turn off the keyboard. :confused: 

Homework is the core part of this class. Through the homework assignments you'll create real apps that you should be proud to show anybody. The more time you spend, the better your portfolio will be and the more you will learn. Try to spend a little more time next week!

**Comment by jackynguyen110** [comment]
Thanks for your helpful advice.I will try to best next assignment for better result. :)
